ui/src/app/edge/index/widget/production/production.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container *ngIf="service.currentEdge | async as edge">
+    <ng-container *ngIf="edge.currentData | async as currentData">
+        <ion-card>
+            <ion-item color="light">
+                <ion-img style="width:40px;" src="assets/img/production.png">
+                </ion-img>
+                <ion-label translate>General.Production</ion-label>
+            </ion-item>
+            <ion-card-content *ngIf="currentData.summary.production as sum">
+                <ion-grid no-padding>
+                    <table class="full_width">
+                        <!-- DC -->
+                        <tr *ngIf="sum.activePowerDC != null">
```

Comment:
Sollten wir in den neuen Widgets dann evtl. auch nur die Gesamt-Produktion ("_sum/ProductionActivePower") anzeigen - den Rest sieht man dann ja im Modal.

Das sollten wir dann aber erst umsetzen, wenn wir auch das Modal live schalten; damit keine Beschwerden kommen.